turbo/engineapi/fork_validator.go

```diff
@@ -228,7 +229,6 @@ func (fv *ForkValidator) ValidatePayload(tx kv.RwTx, header *types.Header, body
 	currentHash := header.ParentHash
 	foundCanonical, criticalError = rawdb.IsCanonicalHash(tx, currentHash)
 	if criticalError != nil {
-		fmt.Println("critical")
 		return
```

Comment:
hiding errors is not good, at least let's add a warning log here

Reply:
that is a debug println left there for accident